src/reactor/poll_evented.rs

```diff
@@ -181,7 +176,7 @@ impl<E> PollEvented<E> {
     pub fn poll_ready(&self, mask: Ready) -> Async<Ready> {
         let bits = super::ready2usize(mask);
         match self.readiness.load(Ordering::SeqCst) & bits {
-            0 => {}
+            0 => {},
```

Comment:
Could the commas here be left out? Or is this inserted by rustfmt? (if inserted by rustfmt can we hold off on that?)